kolibri/core/assets/src/views/BottomNavigationBar.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+
+  <div class="bottom-bar" :style="{ backgroundColor: $themeTokens.textInverted }">
+    <span v-for="(routeDefinition, key) in bottomMenuOptions[0].routes" :key="key">
+      <a
```

Comment:
Maybe I am misunderstanding - but isn't the BottomNavigationBar not always showing links for the currently active SPA? Could we not use VueRouter then, rather than the generateNavRoute?

Reply:
it is not. in the current design - it is always showing the learn routes. I was supposed to follow up with Jessica on this and forgot. will ask in slack now